tests/nlu/test_train.py

```diff
@@ -90,6 +90,8 @@ def test_all_components_are_in_at_least_one_test_pipeline():
 
 
 @pytest.mark.parametrize("language, pipeline", pipelines_for_tests())
+@pytest.mark.linux
+@pytest.mark.darwin
```

Comment:
some of these pipelines should work on Windows. Can we be more selective instead of skipping to run them all?